docs/install.rst

```diff
@@ -70,122 +97,150 @@ by executing the following command in your conda env:
+                ENV RMQ_HOSTNAME=two.radical-project.org
+                ENV RMQ_PORT=33247
+                ENV RADICAL_PILOT_DBURL="mongodb://user:user@ds247688.mlab.com:47688/entk-docs"
 
-        docker build -f ./Dockerfile -t entk .
-        docker run -t -i entk
+                RUN apt-get update \
+                && apt-get install wget curl python python-dev python-pip python-virtualenv bzip2 -y \
+                && virtualenv ~/ve-entk \
+                && . ~/ve-entk/bin/activate \
+                && pip install radical.entk
 
-Once you execute the container, the default path will be /root (of the container).
-The EnTK virtualenv exists at ~/ve-entk (inside the container). This is useful
-to know as the examples exist inside the virtualenv.
+        You can also download the Dockerfile :download:`here <./misc/Dockerfile>`.
 
-You can check the version of Ensemble Toolkit with the
-```radical-entk-version``` command-line tool. The current version should be
-printed.
+        You can build and execute the container by running:
 
-.. code-block:: bash
+        .. code-block:: bash
 
-        radical-entk-version
-        0.70.0
+                docker build -f ./Dockerfile -t entk .
+                docker run -t -i entk
 
+        Once you execute the container, the default path will be /root (of the container).
+        The EnTK virtualenv exists at ~/ve-entk (inside the container). This is useful
+        to know as the examples exist inside the virtualenv.
 
-Installing rabbitmq
-===================
+        You can check the version of Ensemble Toolkit with the
+        ```radical-entk-version``` command-line tool. The current version should be
+        printed.
 
-Installing rabbitmq as a system process (sudo privileges required)
-------------------------------------------------------------------
+        .. code-block:: bash
 
-Ensemble Toolkit relies on RabbitMQ for message transfers. Installation
-instructions can be found at <https://www.rabbitmq.com/download.html>. At
-the end of the installation run ```rabbitmq-server``` to start the server.
-RabbitMQ needs to be installed on the same machine as EnTK is installed.
+                radical-entk-version
+                0.70.0
 
-In some cases, you might have to explicitly start the rabbitmq-server after
-installation. You can check if the rabbitmq-server process is alive. If not,
-please run the following:
+
+RabbitMQ
+========
+
+Ensemble Toolkit relies on RabbitMQ for message transfers. RabbitMQ needs to be
+configured or it can be installed on the same machine as EnTK is installed.
+Installation instructions can be found at
+<https://www.rabbitmq.com/download.html>. At the end of the installation run
+```rabbitmq-server``` to start the server.
+
+The following configuration defines a default server and port number to communicate.
 
 .. code-block:: bash
 
-        rabbitmq-server -detached
+        export RMQ_HOSTNAME=two.radical-project.org; export RMQ_PORT=33239
 
```

Comment:
I would anonymize this, especially because at the moment we do not password-protect it. Something like:
```
         export RMQ_HOSTNAME=<hostname>; export RMQ_PORT=port
```
I would also add documentation about password protected hosts. See Sean patch and relative documentation.